### PR TITLE
added projection terms

### PR DESCRIPTION
**Pull request by shawntanzk**
Notes:
I did not add references as the name describes what they are - if need refs of existence, might all be the DOI of the flagship paper (https://www.biorxiv.org/content/10.1101/2020.10.19.343129v1)

I've covered all the projection type we need except of long range projecting which I think is not useful as a PATO term. It refers specifically to interneurons that project long range rather than nearby but I think that's tricky to put as a PATO term

### CODE REVIEW

**Review by patrick-lloyd-ray — DISMISSED** [review]
Looks good to me.

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -22043,6 +22043,49 @@ is_a: PATO:0001655 ! osmolarity
 created_by: http://orcid.org/0000-0002-3410-4655
 creation_date: 2020-10-30T08:08:48Z
 
+[Term]
+id: http://purl.obolibrary.org/obo/PATO0070027
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:22047]: ID syntax

File: src/ontology/pato-edit.obo [hunk 2]
```diff
@@ -22043,6 +22043,49 @@ is_a: PATO:0001655 ! osmolarity
...
+name: intratelencephalic projection
+def: "A prolongation or process extending from a cell derived from the telencephalon to structures derived from the telencephalon." []
+synonym: "IT projection" EXACT []
+xref: ILX:0770100
+is_a: GO:0042995 ! cell projection
+property_value: http://purl.org/dc/elements/1.1/creator http://orcid.org/0000-0001-7258-9596
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:22053]: created_by

File: src/ontology/pato-edit.obo [hunk 3]
```diff
@@ -22043,6 +22043,49 @@ is_a: PATO:0001655 ! osmolarity
 created_by: http://orcid.org/0000-0002-3410-4655
 creation_date: 2020-10-30T08:08:48Z
 
+[Term]
+id: http://purl.obolibrary.org/obo/PATO0070027
+name: intratelencephalic projection
+def: "A prolongation or process extending from a cell derived from the telencephalon to structures derived from the telencephalon." []
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:22049]: A neuron projection quality that is a characteristic of neurons with both soma and projections entirely within the telencephalon.

**Review by dosumis — CHANGES_REQUESTED** [review]
Add a general class 'neuron projection quality' under 'physical quality'
def: "A physical quality inhering in a neuron by virtue of the relative location of its soma and the regions to which it projects and makes synaptic contacts in."

Use this in place of current GO term.

See also notes on defs & IDs.

**Review by patrick-lloyd-ray — APPROVED** [review]

File: src/ontology/pato-edit.obo [hunk 4]
```diff
@@ -22035,6 +22035,56 @@ xref: NLX:378
...
+def: "A neuron projection quality that is a characteristic of neurons with projections that terminate locally rather than distantly." [DOI:10.1101/2020.10.19.343129]
+synonym: "NP" EXACT []
+xref: ILX:0770103
+is_a: PATO:0070033 ! neuron projection quality
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070031
+name: medulla projection
+def: "A neuron projection quality that is a characteristic of neurons with projections that terminate in the medulla." [DOI:10.1101/2020.10.19.343129]
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:22067]: medulla ->  medulla oblongata ?

There are lots of medullas in anatomy, I think above is the correct one.

I also think that a tighter definition would be better "neurons that project to the medulla oblongata from other brain regions".   (For our purposes coritical medulla projecting might be even better - but we would need to change the term name).

GCI: bearer_of some 'medulla projecting' subClassOf  'axon synapses in' some 'medulla oblongata'

File: src/ontology/pato-edit.obo [hunk 5]
```diff
@@ -22035,6 +22035,56 @@ xref: NLX:378
...
+id: PATO:0070033
+name: neuron projection quality
+def: "A physical quality inhering in a neuron by virtue of the relative location of its soma and the regions to which it projects and makes synaptic contacts in." [DOI:10.1101/2020.10.19.343129]
+is_a: PATO:0001018 ! physical quality
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070034
+name: intratelencephalic projection
+def: "A neuron projection quality that is a characteristic of neurons with both soma and projections entirely within structures derived from the telencephalon." [DOI:10.1101/2020.10.19.343129]
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/pato-edit.obo:22082]: GCI: 'bearer of' some 'intratelencephalic projection' subClassOf 'part of' some telencephalon

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments.   Let me know if you need help with adding GCIs.

File: src/ontology/pato-edit.obo (hunk 4 above)
Comment (shawntanzk) [reply src/ontology/pato-edit.obo:22067]: yes, its the medulla oblongata, my bad, neuroscientists assumptions/bad habits to equate medulla to medulla oblongata i guess :)

**Comment by shawntanzk** [comment]
GCIs will be addressed in another PR (see issue #418)

**Comment by shawntanzk** [comment]
> GCIs will be addressed in another PR (see issue #418)

Fixed it in this PR instead

**Comment by shawntanzk** [comment]
@dosumis think I fixed everything and added GCIs right and all, would like a review, thanks

**Review by dosumis — APPROVED** [review]

**Pull request merged**